server/routes/api/webhookSubscriptions.ts

```diff
@@ -0,0 +1,98 @@
+import Router from "koa-router";
+import { compact } from "lodash";
+import { ValidationError } from "@server/errors";
+import auth from "@server/middlewares/authentication";
+import { WebhookSubscription, Event } from "@server/models";
+import { authorize } from "@server/policies";
+import { presentWebhookSubscription } from "@server/presenters";
+import { WebhookSubscriptionEvent } from "@server/types";
+import { assertArray, assertPresent, assertUuid } from "@server/validation";
+import pagination from "./middlewares/pagination";
+
+const router = new Router();
+
+router.post("webhookSubscriptions.list", auth(), pagination(), async (ctx) => {
+  const { user } = ctx.state;
+  authorize(user, "listWebhookSubscription", user.team);
+  const webhooks = await WebhookSubscription.findAll({
+    where: {
+      createdById: user.id,
```

Comment:
```suggestion
      teamId: user.teamId,
```

Suggest we list all webhooks in the team to allow any admin to manage?

Reply:
Ahh yup that makes more sense, especially since we already changed the authorize layer to let any admin through!